products/llm_analytics/frontend/text-view/constants.ts

```diff
@@ -3,7 +3,7 @@
  */
 
 /** Timeout for text representation API requests in milliseconds */
-export const TEXT_REPR_API_TIMEOUT_MS = 10000 // 10 seconds
+export const TEXT_REPR_API_TIMEOUT_MS = 30000 // 30 seconds
```

Comment:
Waiting 30 seconds is not an acceptable experience, even 10 seconds is long, but okay for a timeout. What makes it take up to 30 seconds?